### PR TITLE
Switch to a relative path within the HTML report output

### PR DESCRIPTION
**Pull request by JamieEdge**
Currently the HTML report output uses the full path to each source file. In the case of using this in a CI environment, the path can be quite long and adds a lot of noise to the report.

This change displays the file paths relative to the directory that SwiftLint was run within (only for HTML). An example of the output is below.

Relative Path Example

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,7 +19,10 @@
 
 * `operator_usage_whitespace` rule is now correctable.  
   [Marcelo Fabri](https://github.com/marcelofabri)
-  
+
+* HTML reports now display a relative path to files.
```
Comment (marcelofabri) [inline CHANGELOG.md]: this is missing 2 trailing whitespaces

Comment (JamieEdge) [inline CHANGELOG.md]: Sorry - fixed now.

**Comment by JamieEdge** [comment]
Thanks - conflicts now resolved.

**Comment by marcelofabri** [comment]
Thanks 🚢

**Pull request merged**